.github/workflows/continuous-integration.yaml

```diff
@@ -40,10 +40,12 @@ jobs:
           - '7.4'
           - '8.1'
           - '8.2'
+          - '8.3'
         symfony:
           - '4.4.*'
           - '5.4.*'
           - '6.2.*'
```

Comment:
All references to Symfony `6.2` may be updated to `6.3` as `6.2` is unmaintained [since july 2023](https://symfony.com/releases/6.2)